media-api/app/lib/Config.scala

```diff
@@ -34,6 +34,7 @@ object Config extends CommonPlayAppConfig with CommonPlayAppProperties {
 
   val imageBucket: String = properties("s3.image.bucket")
   val thumbBucket: String = properties("s3.thumb.bucket")
+  val dataStoreBucket: String = properties("s3.datastore.bucket")
```

Comment:
No, but I want to get something to test infront of users, while we're doing that I can start setting up another service.